<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/downloadtoolbarwidget.ui -->
<!-- Phiên bản: 1.2 (Dọn dẹp giao diện) -->
<ui version="4.0">
 <class>DownloadToolbarWidget</class>
 <widget class="QWidget" name="DownloadToolbarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Loại:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="downloadTypeComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Định dạng:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="formatComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Chất lượng:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="qualityComboBox"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Lưu tại:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="savePathComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="getFromLibraryCheckBox">
     <property name="text">
      <string>Lấy link từ Thư viện</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="settingsButton">
     <property name="text">
      <string>⚙️</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="helpButton">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

